core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -80,6 +80,49 @@ var ExplorationPlayerPage = function() {
   var suggestionPopupLink =
     element(by.css('.protractor-test-exploration-suggestion-popup-link'));
 
+  var audioBar = element(
+    by.css('.protractor-test-audio-bar'));
+  var audioSelect = element(
```

Comment:
Generally we try and keep variables named as nouns. Maybe `audioSelector`?

Reply:
changed!